packages/eas-cli/src/commands/update/index.ts

```diff
@@ -601,13 +621,18 @@ async function getRuntimeVersionObjectAsync(
   }
 
   return Object.fromEntries(
-    platforms.map(platform => [
-      platform,
-      nullthrows(
-        Updates.getRuntimeVersion(exp, platform),
-        `Unable to determine runtime version for ${platform}`
-      ),
-    ])
+    platforms.map(platform => {
+      if (platform === 'web') {
+        return ['web', 'n/a'];
```

Comment:
```suggestion
        return ['web', null];
```
?

Reply:
Agree that using null is a good idea -- if there's a special sentinel value we need to check for, it helps to make the type checker aware of it, and `null` is the most built-in way to do that